.circleci/config.yml

```diff
@@ -381,6 +347,21 @@ workflows:
                 - development
     jobs:
       - build
-      - functional_js_tests__config_only:
+      - dev_build:
```

Comment:
D'oh, this didn't quite catch my marketing skills. I started with a 'dev' build but as that's quite ambiguous, I preferred to stick with 'edge build'. Probably not worth changing here unless it really bugs or confuses someone.

Reply:
It bugged me so I fixed it :)